index.html

```diff
@@ -0,0 +1,164 @@
+    </head>
+    <body>
+    <main>
+       <header>
+        <nav class="navigation">
+            <div class="hamburger">
+                <span class="ham"></span>
+                <span class="ham"></span>
+                <span class="ham"></span>
+            </div>
+            <ul class="menu">
+                <li class="menu-items">
+                    <a href="./index.html">Home</a>
+                </li>
+                <li class="menu-items">
+                    <a href="./about.html">About</a>
+                </li>
+                <li class="menu-items">
+                    <a href="#">News</a>
+                </li>
+                <li class="menu-items">
+                    <a href="#">Contact</a>
+                </li>
+            </ul>
+            <div class="desktop-menu">
+                <ul class="nav-before">
+                    <li>
+                        <a href="#"><i class="bi bi-facebook"></i></a>
+                    </li>
+                    <li>
+                        <a href="#"><i class="bi bi-twitter"></i></a>
+                    </li>
+                    <li class="nav-icons">
+                        <a href="#" id="lang">English</a>
+                    </li>
+                    <li class="nav-icons">
+                        <a href="#">My Page</a>
+                    </li>
+                    <li class="nav-icons">
+                        <a href="#">Logout</a>
+                    </li>
+                </ul>
+                <ul class="desktop-menu-links">
+                    <li>
+                        <a href="./index.html">
+                         <img src="./icons/lg01.png" id="nav-logo" alt="cc logo">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="./about.html">About</a>
+                    </li>
+                    <li>
+                        <a href="#about">Program</a>
+                    </li>
+                    <li>
+                        <a href="#">Live training</a>
+                    </li>
+                    <li>
+                        <a href="#">Sponsors</a>
+                    </li>
+                    <li>
+                        <a href="#">Web3 careers</a>
+                    </li>
+                    <li id="camp">
+                        <a href="#">CC Campaign</a>
+                    </ul>
+            </div>
+        </nav>
+       </header>
+        <section id="home" class="home-page">
+            <h1 class="headline"><span id="intro">Reorient, Rethink, Revolutionize</span>WELCOME<br>TO BLOCKCHAIN WORLD <br> AND NFT'S</h1>
+            <p class="summit-description">
+                Embrace Web3 with a range of certifications offered by the Blockchain Council, 

```

Comment:
- [ ] As requested by the [previous reviewer](https://github.com/Oscah01/Blockchain-council/pull/2#discussion_r938362192), kindly give your elements the correct margins and padding to match the provided Behance design. 
- [ ] The date "Mon, 01st August 2022" is overlapping the rest of the text. Kindly fix this.

**This is how your design currently looks like:.**

![image](https://user-images.githubusercontent.com/86318284/183051948-b2db5100-4c8a-4416-b8a8-919d7e7b2d98.png)